src/payment/strategies/ppsdk/sub-strategies/card-sub-strategy.ts

```diff
@@ -0,0 +1,57 @@
+import { CheckoutStore } from '../../../../checkout';
+import { InvalidArgumentError, NotInitializedError, NotInitializedErrorType } from '../../../../common/error/errors';
+import { HostedForm, HostedFormFactory } from '../../../../hosted-form';
+import { OrderActionCreator } from '../../../../order';
+import { PaymentArgumentInvalidError } from '../../../errors';
+import { PaymentInitializeOptions } from '../../../payment-request-options';
+import { SubStrategy, SubStrategySettings } from '../ppsdk-sub-strategy';
+
+export class CardSubStrategy implements SubStrategy {
+    protected _hostedForm?: HostedForm;
+
+    constructor(
+        private _store: CheckoutStore,
+        private _orderActionCreator: OrderActionCreator,
+        private _hostedFormFactory: HostedFormFactory
+    ) {}
+
+    async execute({ payment }: SubStrategySettings) {
```

Comment:
It's a good practice to write out the return type of a public method so it's clear to other engineers reading the code.